### PR TITLE
Install and configure rubocop

### LINKED ISSUE #7: Install and configure rubocop

### PR DESCRIPTION
**Pull request by erbdevs**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ services:
 
   web:
     <<: *app_config
-    container_name: simba_web
```
Comment (pauc) [inline docker-compose.yml:36]: 😁 🤷‍♀️, sorry

**Comment by pauc** [comment]
👏 👏 👏 

Now we need to fix the linter warnings to make all "checks" pass, isn't it?

**Review by pauc — APPROVED** [review]

**Comment by pauc** [comment]
Did you try to reuse the same workfow we are using for rspec tests? Only curious... Do you know what's the difference/implications of reuse the same action with different steps or use different actions?

**Comment by erbdevs** [comment]
> Did you try to reuse the same workfow we are using for rspec tests? Only curious... Do you know what's the difference/implications of reuse the same action with different steps or use different actions?

Not yet. I am still trying different things to get what I want with this new workflow. Ill tell you. when I get more information :)

**Comment by erbdevs** [comment]
> 👏 👏 👏
> 
> Now we need to fix the linter warnings to make all "checks" pass, isn't it?

Correct, and It is fixed now to pass only if we have not offenses in ruby changed files. Take a look at the latter commits

**Pull request merged**